test/bandit_test.py

```diff
@@ -0,0 +1,335 @@
+import pytest
+
+from eppo_client.sharders import MD5Sharder, DeterministicSharder
+
+from eppo_client.bandit import (
+    ActionContext,
+    Attributes,
+    score_numeric_attributes,
+    score_categorical_attributes,
+    BanditEvaluator,
+)
+from eppo_client.models import (
+    BanditCoefficients,
+    BanditModelData,
+    BanditNumericAttributeCoefficient,
+    BanditCategoricalAttributeCoefficient,
+)
+
+bandit_evaluator = BanditEvaluator(MD5Sharder(), 10_000)
```

Comment:
👏 Love the explicit tests for the evaluator! 🎉 

I had some rediculous spreadsheet for my top-level eppo client tests 🤦‍♂️